lib/note-detail.jsx

```diff
@@ -107,13 +99,17 @@ export default React.createClass( {
 	},
 
 	renderEditable( divStyle ) {
-		const note = this.props.note;
+		const { content = '' } = this.props.note.data;
```

Comment:
ah you're right, I was looking at the wrong place. should I change this both in `renderMarkdown` and `renderEditable` then?